src/gravel/controllers/nodes/messages.py

```diff
@@ -41,6 +41,7 @@ class WelcomeMessageModel(BaseModel):
     pubkey: str
     cephconf: str
     keyring: str
+    peer_url: str
```

Comment:
Maybe I'm just nitpicking, but can we give this a slightly different name? Because it's peer_url, it makes me think it's a URL, but it's actually a string in the form "name=URL". Maybe call it, ummm... etcd_peer? Just something to give a little more information.

Reply:
(oops, that comment should have been part of the review below)

Reply:
yep, sounds good.